creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -414,11 +414,36 @@ At this time, `bridge` mode doesn’t support additional configuration
 fields.
 
 ______________________________________________________________________________
-*Note:* due to IPv4 address delagation, in `bridge` mode the pod doesn’t have
+*Note:* due to IPv4 address delegation, in `bridge` mode the pod doesn’t have
 an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* there is an option to disable bridge interface with pod network.
+This is done with a designated flag.
+The default value of the flag is 'true', meaning bridge interface is enabled.
+To disable it change flag value to 'false'. For example:
+______________________________________________________________________________
+apiVersion: v1
```

Comment:
you can do something like this https://github.com/kubevirt/user-guide/pull/267/files#diff-5ac883b57d09207b7ff7a1dae3413f18L399 so the code will look better